tests/foreman/cli/test_errata.py

```diff
@@ -1534,3 +1535,43 @@ def test_errata_list_by_contentview_filter(module_sca_manifest_org, module_targe
         )
     )
     assert errata_count != errata_count_cvf
+
+
+@pytest.mark.rhel_ver_match('8')
```

Comment:
Is this only applicable on RHEL8? If not consider using default rhel version in content_host.yaml

Reply:
It's not version specific.
Any rhel host can be used, as only need to run `errata recalculate` command on that host which has applicable errata's to start `"Actions::Katello::Applicability::Hosts::BulkGenerate"` task and expecting `worker-hosts-queue-1.service` to be proceed with it's thread execution.